/* response_generic.proto/Open GoPro, Version 1.0 (C) Copyright 2021 GoPro, Inc. (http://gopro.com/OpenGoPro). */
/* This copyright was auto-generated on Tue May 18 22:08:50 UTC 2021 */

/*
Defines the structure of preset status protobuf messages from the camera
*/

syntax="proto2";
package open_gopro;


message ResponseGeneric{
    required EnumResultGeneric result = 1;
}


enum EnumResultGeneric{
    RESULT_UNKNOWN = 0;
    RESULT_SUCCESS = 1;
    RESULT_ILL_FORMED = 2;
    RESULT_NOT_SUPPORTED = 3;
    RESULT_ARGUMENT_OUT_OF_BOUNDS = 4;
    RESULT_ARGUMENT_INVALID = 5;
}

